apps/graph/base.de.i18n

```diff
@@ -31,3 +31,11 @@ DerivativeFunctionColumn = "Spalte der Ableitungsfunktion"
 HideDerivativeColumn = "Ableitungsfunktion ausblenden"
 AllowedCharactersAZaz09 = "Erlaubt: A..Z, a..z, 0..9, _"
 ReservedName = "Reserviertes Wort"
+FunctionColorBlack = "schwarz"
+FunctionColorBlue = "blau"
+FunctionColorRed = "rot"
+FunctionColorPurple = "lila"
+FunctionColorGreen = "grün"
+FunctionColorOrange = "Orange"
+FunctionColorYellow = "gelb"
+FunctionColorWhite = "weiß"
```

Comment:
Add carriage return (for all files which don't have it)

Reply:
okay i do that